src/components/table/examples/table-layout-fitDataStretch.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { invoices, Invoice } from './invoices';
+
+/**
+ *
+ * ````tsx
+ * layout="fitDataStretch"
+ * ````
+ *
+ * Works just like `fitDataFill`, but unlike the default layout
+ * which resulted in having an empty last column, in this layout
+ * the last existing column will stretch out to fill up the remaining table width.
```

Comment:
> The last existing column will stretch out to fill up the remaining table width. 

- do you mean the last filled column? or this empty column should not be here?

Reply:
yes, last existing = last filled, or last column with content
consumers send an array of columns to tables, so the last one of them.